rpm-spec/nvidia-xrun.spec

```diff
@@ -1,36 +1,48 @@
+%define nvidia_xrun_pm_version 0.3.2
 Name: nvidia-xrun-pm
-Version: 20181128
+Version: %{nvidia_xrun_pm_version}
 Release: 1%{?dist}
 Summary: Utility to run separate X with discrete nvidia graphics with full performance.
 License: GNU GPLv2
-Source0: nvidia-xrun
-Source1: nvidia-xorg.conf
-Source2: nvidia-xinitrc
-Source3: nvidia-xrun-pm.service
 buildroot: %{_tmppath}/%{name}-root
 BuildArch: noarch
 Conflicts: nvidia-xrun
+# The "systemd" package is required for the %{_unitdir} macro.
+BuildRequires: git systemd
 
 %description
 This fork provides the benefits of nvidia-xrun without the extra dependency of bumblebee for power management. These utility scripts aim to make the life easier for nvidia cards users. It started with a revelation that bumblebee in current state offers very poor performance. This solution offers a bit more complicated procedure but offers a full GPU utilization.
 
+%build
+rm -rf nvidia-xrun-pm
+# SSL verification is turned off to temporarily workaround openSUSE Leap build issues on COPR.
+git -c http.sslVerify=false clone --depth 1 --single-branch --branch %{nvidia_xrun_pm_version}\
```

Comment:
Wouldn't better to avoid git as dependency by downloading the archive at `https://github.com/michelesr/nvidia-xrun-pm/archive/${nvidia_xrun_pm_version}.tar.gz`? Or you could provide the package directly to the build script so that you don't need to avoid SSL verification (at the moment SSL verification is disabled and there's no checksum validation)

Reply:
There was something wrong with how the certificate authorities of openSUSE Leap are setup on COPR leading to a complaint about GitHub's SSL not being valid (which the certificate is obviously valid and correct). The build worked fine with openSUSE Tumbleweed.

I will update the patch to use a URL source of the release package tarball and see if there are any SSL verification issues with that.

The way the RPM spec was handled before allowed us to package the files without downloading anything. This had the disadvantages of (1) you had to manually copy the correct files to `~/rpmbuild/SOURCES/` and (2) there is no guarantee if someone cloned the repository and is on the master branch or properly downloaded/checked-out a release tag. A person on the master branch could build the RPM spec file designed for 0.3.2 but in reality they are making an RPM for 0.3.2 + 6 commits, for example.